bin/registry-server-start.sh

```diff
@@ -47,6 +47,8 @@ if [ ! "x$EXT_CLASSPATH" = "x" ]; then
  CLASSPATH=$CLASSPATH:$EXT_CLASSPATH;
 fi
 
+CLASSPATH=$CLASSPATH:$base_dir/hadoop-conf/
```

Comment:
core-site.xml should be copied to this folder during SR installation. _core-site.xml_ should be in the classpath for correct UGI initialization.